test/libponyc-run/ffi-libponyc-standalone/main.pony

```diff
@@ -15,7 +15,7 @@ type TokenId is I32
 actor Main
   new create(env: Env) =>
     try
-      ifdef linux or osx or windows then
+      ifdef freebsd or linux or osx or windows then
```

Comment:
see aforementioned comment about switching to "not" logic as there are fewer "nots" than "yeses" now.